server/repositories/cmsQueries/exploreContentQuery.js

```diff
@@ -0,0 +1,45 @@
+const { DrupalJsonApiParams: Query } = require('drupal-jsonapi-params');
+const { getSmallTile } = require('../../utils/jsonApi');
+
+class ExploreContentQuery {
+  static #TILE_FIELDS = [
+    'drupal_internal__nid',
+    'title',
+    'field_moj_thumbnail_image',
+    'field_thumbnail_image',
```

Comment:
Hasn't `field_thumbnail_image` been removed?

Reply:
Looks like it needs removing from `server/repositories/cmsQueries/recentlyAddedContentQuery.js` and probably `server/repositories/cmsQueries/homePageQuery.js`. Also there specs 🤔

Reply:
updated